### PR TITLE
prepare for pub and v1.0.0 release

### PR DESCRIPTION
**Pull request by elizabethmcd**
Adds workflow diagram to README, cleans some things up for pub release (addressed in #60), CITATION.cff removed since we refer to our pubs in READMEs and having this ready for v1 creates chicken/egg problems with pub release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 **Arcadia-Science/metagenomics** is a pipeline for profiling metagenomes obtained through either Illumina or Nanopore technologies.
 
+![](img/workflow-overview.jpg)
+
```
Comment (taylorreiter) [inline README.md:14]: ooh pretty yay

Comment (taylorreiter) [reply README.md:14]: sorry just looked at the figure -- i think `taxannotate` should be `tax annotate` right? or is there an entry point with no space in it?

**Review by taylorreiter — APPROVED** [review]
LGTM! one comment but not too concerned about it.

**Review by mertcelebi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (elizabethmcd) [reply README.md:14]: hmm the module is called taxannotate so I left it that way for clarity?

Comment (taylorreiter) [reply README.md:14]: oh that makes total sense! perfect!

**Pull request merged**